<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New_Tag</class>
 <widget class="QWidget" name="New_Tag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый тег</string>
  </property>
  <widget class="QLineEdit" name="new_tag_text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="new_tag_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название тега:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
